### ISSUE TITLE
Repository filters seem to change randomly

### ISSUE THREAD

**Issue by shani117** [rstedman/tfs-pullrequest-dashboard#59]
The repository filters selected to define my view suddenly changes overnight and a lot more repositories are selected in my view without me actually having selected those repositories.

**Comment by cmosnick** [comment]
This is a long running issue for me as well.

**Comment by ScottLangham** [comment]
Yep, in my organization we have many teams. Any time another team creates a new repo, it gets included on my dashboard.

I often have to spend carefully scroll through a huge list of repos trying to find any new ones so that I can uncheck them to remove them again.

It would be nice if new repos were not automatically included on my dashboard.
